app/views/browse/mobile-clients.html

```diff
@@ -0,0 +1,65 @@
+<div class="middle">
+  <div class="middle-header">
+    <div class="container-fluid">
+      <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+      <alerts alerts="alerts"></alerts>
+      <div>
+        <h1 class="contains-actions">
+          <div class="pull-right dropdown" ng-if="ctrl.mobileClient">
+            <button type="button" class="dropdown-toggle btn btn-default actions-dropdown-btn hidden-xs" data-toggle="dropdown">
+              Actions
+              <span class="caret" aria-hidden="true"></span>
+            </button>
+            <a href=""
+               class="dropdown-toggle actions-dropdown-kebab visible-xs-inline"
+               data-toggle="dropdown"><i class="fa fa-ellipsis-v"></i><span class="sr-only">Actions</span></a>
+            <ul class="dropdown-menu dropdown-menu-right actions action-button">
+              <li>
+                <delete-link
+                  kind="MobileClient"
+                  group="mobile.k8s.io"
+                  resource-name="{{ctrl.mobileClient.metadata.name}}"
+                  project-name="{{ctrl.mobileClient.metadata.namespace}}"
+                  alerts="alerts"
+                  redirect-url={{ctrl.redirectUrl}}>
+                </delete-link>
+              </li>
+            </ul>
+          </div>
+          {{ctrl.mobileClient.spec.name}}
+          <small class="meta" ng-if="ctrl.mobileClient">created <span am-time-ago="ctrl.mobileClient.metadata.creationTimestamp"></span></small>
+        </h1>
+        <labels labels="ctrl.mobileClient.metadata.labels" clickable="true" kind="mobileClient" project-name="{{ctrl.mobileClient.metadata.namespace}}" limit="3"></labels>
+      </div>
+    </div>
+  </div>
+  <div class="middle-content" persist-tab-state>
+    <div class="container-fluid">
+      <div ng-if="!ctrl.loaded">Loading...</div>
+      <div class="row">
+        <div class="col-md-12" ng-class="{ 'hide-tabs' : !ctrl.mobileClient }">
+          <uib-tabset>
+            <uib-tab>
+              <uib-tab-heading>Configuration</uib-tab-heading>
+              <pre>{{ctrl.mobileClient | json}}</pre>
+            </uib-tab>
+            <uib-tab>
+              <uib-tab-heading>SDK Setup</uib-tab-heading>
+              <div ng-switch="ctrl.mobileClient | mobileClientType">
+                <div ng-switch-when="iOS">
+                  <ng-include src="'views/mobile-client-sdk-config/sdk-config-ios.html'"></ng-include>
+                </div>
+                <div ng-switch-when="android">
+                  <ng-include src="'views/mobile-client-sdk-config/sdk-config-android.html'"></ng-include>
+                </div>
+                <div ng-switch-default>
+                  <ng-include src="'views/mobile-client-sdk-config/sdk-config-default.html'"></ng-include>
```

Comment:
I would leave this inline. It will be a small static fallback which helps readability of the view